app/build.gradle

```diff
@@ -262,16 +262,31 @@ android.applicationVariants.all { variant ->
 
         println("versionName override: $versionName")
 
-        variant.outputs.each { output ->
-            def abi = output.getFilter(OutputFile.ABI)
-            // We use the same version code generator, that we inherited from Fennec, across all channels - even on
-            // channels that never shipped a Fennec build.
-            def versionCodeOverride = Config.generateFennecVersionCode(abi)
 
-            println("versionCode for $abi = $versionCodeOverride")
+        if (project.hasProperty("mozillaOnline") || gradle.hasProperty("localProperties.mozillaOnline")) {
+            variant.outputs.each { output ->
+                def abi = output.getFilter(OutputFile.ABI)
+                // We use the same version code generator, that we inherited from Fennec, across all channels - even on
+                // channels that never shipped a Fennec build.
+                def versionCodeOverride = Config.generateFennecVersionCode("armeabi-v7a")
```

Comment:
It looks like this line is the only one that it is changing between `if` and `else` block. We could have variable and changes it's value to `abi`  or `"armeabi-v7a"` depending on if `project.hasProperty("mozillaOnline") || gradle.hasProperty("localProperties.mozillaOnline")` is `true`